aii-opennebula/src/main/pan/quattor/aii/opennebula/schema.pan

```diff
@@ -188,6 +188,8 @@ type opennebula_vmtemplate = {
     "ignoremac" ? opennebula_ignoremac
     "graphics"  : string = 'VNC' with match (SELF, '^(VNC|SDL|SPICE)$')
     "diskcache" ? string with match(SELF, '^(default|none|writethrough|writeback|directsync|unsafe)$')
+    @{specific image mapping driver. qcow2 is not supported by Ceph storage backends}
```

Comment:
Mmm.. that will require some extra work. At this moment to use the opennebula-aii we are mapping the VM hard drives to specific datastores, as example:

```
"datastore" = dict(
     "vda", "my_datastore1",
     "vdb", "my_datastore2");
```
so the aii will try to generate each image in each datastore, but if the datastore does not exist it will show an error message, at this moment does not detect if the datastore exits or what kind of datastore are you requesting (NFS,Ceph..), the aii just tries to use the datastore that are you asking for... so if you want to generate a qcow2 image and your datastore is located in a Ceph storage backend the aii will show an error message in the same way and the image is not generated and the VM is not started.
What we can do is to include a check from in the opennebula-aii logic to detect first of all  if the datastore is there and then what kind of datastore are you asking for to display a nicer error message that you are trying to use the wrong storage type.. I can open a ticket to include this.